cli/src/test/java/com/liferay/blade/cli/TestUtil.java

```diff
@@ -39,6 +48,44 @@
  */
 public class TestUtil {
 
+	public static File checkFileDoesNotExists(Path path) {
```

Comment:
none of these tests are used outside of CreateCommandTEst.  So there is no reason yet to externalize them. Lets leave them in CreateCommandTest.